liquibase-core/src/main/java/liquibase/sqlgenerator/core/CreateViewGenerator.java

```diff
@@ -48,14 +51,23 @@ public ValidationErrors validate(CreateViewStatement createViewStatement, Databa
 
     @Override
     public Sql[] generateSql(CreateViewStatement statement, Database database, SqlGeneratorChain sqlGeneratorChain) {
+        List<Sql> sql = new ArrayList<Sql>();
+        List<String> mssqlSetStatementsBefore = new ArrayList<>();
 
```

Comment:
Can you add property to set default delimiter and use it in this part of code?

Reply:
Done